beacon_node/network/src/sync/range_sync/batch.rs

```diff
@@ -203,6 +207,17 @@ impl<T: EthSpec, B: BatchConfig> BatchInfo<T, B> {
         }
     }
 
+    pub fn outcome(&self) -> OpOutcome {
```

Comment:
We could add docs for this function or change the method name to something more descriptive. Happy to keep it as is though.

Reply:
done in https://github.com/sigp/lighthouse/pull/3439/commits/6869ba426b57905618e98f18fa757e8a194886dc